include/wlcs/display_server.h

```diff
@@ -25,8 +25,9 @@
 extern "C" {
 #endif
 
-typedef struct wl_surface wl_surface;
-typedef struct wl_display wl_display;
+struct wl_surface;
+struct wl_display;
+struct wl_event_loop;
```

Comment:
Why this change of style? Unless there's a good reason I'd recommend not doing this: The original `typedef struct wl_surface wl_surface` (as used elsewhere in the file) introduces `wl_surface`` to both C namespaces, and hides that incompatibility with C++.

Reply:
So, there's an idiom in Wayland to have code like
```
    struct wl_surface* wl_surface;
```
Which IIRC breaks if we have the typedef visible.

Reply:
OK, that's a fair justification. (Although I'd argue it is an "unrelated change" and ought to be done consistently.)